extensions/decimal_arithmetics/description.yml

```
@@ -0,0 +1,28 @@
+extension:
+  name: decimal_arithmetics
+  description: Adds UDFs for high-precision decimal arith
+  version: 1.0.0
+  language: C++
+  build: cmake
+  license: MIT
+  maintainers:
+    - varunkumar
+
+repo:
+  github: varunkumar/duckdb-decimal-arithmetics
```

```suggestion
  github: varunkumar/duck-decimal-arithmetics
```
? Repo name has to match

---

There was a typo earlier. Should the repo name and extension name match?

---

Thanks, looks good to me now. And no, repository name and extension name are not tied.